src/py/api.rs

```diff
@@ -7,15 +7,22 @@ use super::run_program::{
     __pyo3_get_function_deserialize_and_run_program2, __pyo3_get_function_run_chia_program,
     __pyo3_get_function_serialized_length,
 };
-use crate::run_program::STRICT_MODE;
+use crate::dialect::{NO_NEG_DIV, NO_UNKNOWN_OPS};
+use crate::gen::flags::{COND_CANON_INTS, NO_UNKNOWN_CONDS};
+
+pub const MEMPOOL_MODE: u32 = NO_NEG_DIV | COND_CANON_INTS | NO_UNKNOWN_CONDS | NO_UNKNOWN_OPS;
```

Comment:
This seems odd to merge flags across two separate namespaces. Are you sure you really want to do this?

Reply:
not in the long run, just for now